src/SolverAlgorithm.C

```diff
@@ -73,6 +73,9 @@ NGPApplyCoeff::NGPApplyCoeff(EquationSystem* eqSystem)
   if (hasOverset_) {
     iblankField_ = nalu_ngp::get_ngp_field<int>(eqSystem->realm_.mesh_info(), "iblank");
   }
+
+  deviceSumInto_ = Teuchos::RCP<CoeffApplier>(eqSystem->linsys_->get_coeff_applier(), CoeffApplierDestructor(), true);
```

Comment:
See comment above on generic CoeffApplier.

Reply:
This line was also reverted.